### ISSUE TITLE
Authentication Issue in v3 After Latest version

### ISSUE THREAD

**Issue by sagargolu** [elsa-workflows/elsa-core#3835]
@sfmskywalker ,

Today, when I took latest code, I am seeing some issues with the code. We are not able to load instances or workflows. May be this is happened due to team has introduced separate module for Identity. I am attaching some of the screenshot for your references. When I tried to debug, everything is coming from the backend properly but may be there is some issue with the front end. Would you please help us in resolving this?

![image](https://user-images.githubusercontent.com/79979623/227866519-bf6ce2f3-5228-47d7-8aca-c3b17600451a.png)
![image](https://user-images.githubusercontent.com/79979623/227866642-827cf7c0-5ffe-45d6-9126-00e8ec733955.png)
![image](https://user-images.githubusercontent.com/79979623/227866719-e9a2958e-9764-4dc8-88fb-85ad92d48b73.png)

**Comment by sfmskywalker** [comment]
Hi @sagargolu I just tried to reproduce the issue, but it seems to work fine for me. Would it be possible for you to zip your solution and send it to me so I have a ready-made sample to investigate?

**Comment by sagargolu** [comment]
@sfmskywalker , I think, there is some issue in js only and also found below as well. Even when, I am trying to login into the app, it is giving "invalid token" token error.

![image](https://user-images.githubusercontent.com/79979623/227893061-b64e126f-6820-48f0-b196-a789a1790e3d.png)

**Comment by sfmskywalker** [comment]
Just to make sure, can you show me the details of Program.cs that configures the Identity feature? A lot has changed in this area.

**Comment by sfmskywalker** [comment]
Actually, I think there may be a bug when using the `AdminUserProvider` (which I assume is what you're using). It provides a default admin user assigned the "admin" role" - but without configuring that, there' no such role, which means the admin user will have no permissions whatsoever.

I'll push a fix for that shortly.

**Comment by sfmskywalker** [comment]
Using the admin provider, I can now reproduce the issue fully. Fix coming soon.

**Comment by sfmskywalker** [comment]
Build [199](https://github.com/elsa-workflows/elsa-core/actions/runs/4530936179) should fix the issue, but please let me know if otherwise.

**Comment by sagargolu** [comment]
@sfmskywalker , bang on..... now it's working..... Sorry for asking more questions but I am not able to capture or store or get response for HttpRequest Post activity. It was working previously. Would you please check on that side as well? We are again stuck at the workflow execution.

**Comment by sagargolu** [comment]
Closing this issue as this is resolved....

**Comment by pccai** [comment]
Our docker images also need update:
docker run -d -t -i  -e ASPNETCORE_ENVIRONMENT='Development' -p 13000:80 elsaworkflows/elsa-v3:latest

**Comment by pccai** [comment]
admin user cannot login!

**Comment by sfmskywalker** [comment]
@pccai you're right, thanks for reporting! I pushed a fix just now.

**Comment by pccai** [comment]
Thanks! but new v3 images looks like the css style is missing.

**Comment by sfmskywalker** [comment]
You're right, I pushed a fix that is currently being built. Thanks for noticing!
